translator/tests/acceptance/environment.py

```diff
@@ -3,3 +3,4 @@
 
 def before_all(context):
     context.gobgp = GoBGP("gobgp:50051")
+    context.config.setup_logging()
```

Comment:
is this something you created? or a built in logging thing?

Reply:
This is The Way That Behave Does It®. 

Basically, if I understand it, by calling this, behave can then capture the logs that the application sends out (which we then write test cases about)